src/scheduler.js

```diff
@@ -54,10 +55,10 @@ export default async({environment, apiKey, slack}) => {
 
       const currentDateTime = moment();
       const targetDeploymentDateTime = moment(outstandingTask.targetDeploymentDateTime, 'YYYY-MM-DD HH:mm Z');
-      const isLegible = currentDateTime.isAfter(targetDeploymentDateTime);
+      const isScheduledTimePassed = currentDateTime.isAfter(targetDeploymentDateTime);
 
-      // log.info(`Found scheduled flag ${f.key} with targetDeploymentDateTime: ${targetDeploymentDateTime}. IsLegible? ${isLegible}`);
-      return isLegible;
+      log.info(`Found scheduled flag ${f.key} with targetDeploymentDateTime: ${targetDeploymentDateTime}. isScheduledTimePassed? ${isScheduledTimePassed}`);
```

Comment:
This logging string will output ${targetDeploymentDateTime} in unix epoch elapsed format which is probably not what you want. Do you mean to format it in a more readable format?